src/test/java/codeu/model/store/basic/ConversationStoreTest.java

```diff
@@ -59,6 +59,14 @@ public void testIsTitleTaken_false() {
     Assert.assertFalse(isTitleTaken);
   }
 
+  @Test
+  public void testGetTotalConversations() {
+    Integer resultSize = conversationStore.countTotalConversations();
+    Integer expectedResult = 1;
+
+    Assert.assertEquals(expectedResult, resultSize);
```

Comment:
From what it looks like there is only one conversation inside the test instance of this class, is that why your expected result is 1 here? (Same question for the rest of the tests in this pull request)

Reply:
Yes, that's correct!